### ISSUE TITLE
Problem with google fonts

### ISSUE THREAD

**Issue by anjanphukan** [matthieua/WOW#229]
It worked well. But I used Google fonts and after that it started to create an issue. When I refresh the page the texts loads first and then the animation starts again. Can you please help me?

**Comment by caroso1222** [comment]
Are you sure you're writing `wow.init();` inside `$( document ).ready(function(){...})`

**Comment by anjanphukan** [comment]
Hi Carlos, yeah, I used it inside `$( document ).ready(function(){...})`. Do you want to check the files or codes?

**Comment by caroso1222** [comment]
Yeah, show me! How are you loading the Google fonts? I do it via css using the `@import url(https://fonts.googleapis.com/css?family=...:100,300,700,...);` statement and it works like a charm.

**Comment by anjanphukan** [comment]
Sorry, I am not using @import. Can I have your email id, so that I can send you the site credentials privately?
